src/image/image.py

```diff
@@ -1,9 +1,10 @@
-from viam.media.video import ViamImage
+import io
 from typing import Dict, List
+
 import numpy as np
 import torch
 from PIL import Image
-import io
+from viam.media.video import ViamImage
```

Comment:
Should this all be in the same paragraph? I have a slight preference for making the Viam-specific import a separate paragraph from the non-Viam third-party imports.

Reply:
It's just how Ruff do its thing, a bit faster for me to jsut rely on the default code (import formating) action on save